APE0.rst

```diff
@@ -122,18 +122,22 @@ Electing the Coordination Committee
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 A Coordination Committee election is held annually to fill any vacant seats.
 Notice of the election must be posted via the Project’s standard communication
-channels (reaching all active members) at least one month before the election
-begins. The election consists of two phases, each lasting two weeks:
+channels for users, contributors and voting members, at least one month before
+the voting phase begins. The election consists of two phases, each lasting at
+least two weeks:
```

Comment:
This change makes it clear that there doesn't have to be a four week window before nominations open. The objective of this wording is to mandate that nominations should be open for at least two weeks, but to give those running the election more time to process nominations or give notice.

We would recommend using this change to give one weeks notice before nominations open, two weeks of nominations and then one week of admin before the voting starts.